app/src/main/java/one/mixin/android/ui/web/WebFragment.kt

```diff
@@ -1258,6 +1265,11 @@ class WebFragment : BaseFragment() {
         fun playlist(list: Array<String>) {
             playlistAction?.invoke(list)
         }
+
+        @JavascriptInterface
```

Comment:
@over140 Should we add an optional param `conversationId` to this method，then we can close webView and open a conversation with the `close` method.

Reply:
this can be considered later